.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
This files under `.idea/` directory should be removed, shouldn't be?

Reply:
done